Classes/Service/NodeRedirectService.php

```diff
@@ -98,8 +98,9 @@ public function createRedirectsForPublishedNode(NodeInterface $node, Workspace $
         $context = $this->contextFactory->create([
             'workspaceName' => 'live',
             'invisibleContentShown' => true,
-            'dimensions' => $node->getDimensions()
+            'dimensions' => $node->getContext()->getDimensions()
```

Comment:
This fix is absolutely correct because we want to preserve the context dimensions here, using the actual node dimension values means something totally different and could be wrong (e.g. "de", "fr" could be assigned to the node, but is not a valid list of dimension values in any preset).

However this is still not correct because we actually need to iterate over all dimension presets to do this for all node variants. I'd suggest to add this in a follow-up.